.github/workflows/reusable-upgrade-testing.yml

```diff
@@ -37,6 +37,10 @@ on:
         type: 'string'
         default: '5.7'
 
+# Disable permissions for all available scopes by default.
```

Comment:
Looks like we're now disabling permissions for all jobs in this callable workflow, but the `upugrade-testing.yml` file is passing `contents: read`. Unless I'm missing something, that can be changed to `{}`.